src/storage/src/client.rs

```diff
@@ -312,6 +313,28 @@ mod v1 {
         storage_class: String,
         #[serde_as(as = "serde_with::DisplayFromStr")]
         size: u64,
+        component_count: i32,
+        kms_key_name: String,
+        etag: String,
+        restore_token: String,
+        content_encoding: String,
+        content_disposition: String,
+        content_language: String,
+        cache_control: String,
```

Comment:
no change needed: it has always been unclear to me if these are `Option<String>` or `String`.  Maybe the v2 proto specification is clearer?